scripts/schedule_fastq_validation.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+"""
+This script schedules fastq validation for all files in an upload area id
+"""
+
```

Comment:
Consider merging this functionality into uploadctl.  There are similar things there already, e.g. `uploadctl test batch <docker_image> <command>`

Reply:
Going to keep this is a separate script for now but adding ticket to explore this